### PR TITLE
[DM-9137] Configure ESXI for LSST SQuaRE CI infrastructure.

### PR DESCRIPTION
**Pull request by jmatt**
This Ansible deployment configures our ESXi Mac Pro machines used for CI.

* Implement firewall role for sshClient and sshServer services.
* Implement ssh role to configure sshd_config and authorized_keys.
* dev.yml provides quick statuses of ESXi machines and their VMs (through hosts inventory file).

	new file:   .gitignore
	new file:   README.md
	new file:   dev.yml
	new file:   hosts
	new file:   roles/firewall/files/service.xml
	new file:   roles/firewall/tasks/main.yml
	new file:   roles/ssh/files/authorized_keys
	new file:   roles/ssh/files/sshd_config
	new file:   roles/ssh/tasks/main.yml
	new file:   site.yml

### CODE REVIEW

File: roles/ssh/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- name: Upload modified /etc/ssh/sshd_config.
+  copy:
+    src: "{{ role_path }}/files/sshd_config"
+    dest: /etc/ssh/sshd_config
+    owner: root
+    group: root
+    mode: 0444
+
+- name: Add sticky bit to /etc/ssh/sshd_config.
+  command: chmod +t /etc/ssh/sshd_config
```
Comment (jhoblitt) [inline roles/ssh/tasks/main.yml:15]: Why is this needed?

File: roles/ssh/files/sshd_config [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+HostKey /etc/ssh/ssh_host_dsa_key
+
+UsePrivilegeSeparation no
+
+SyslogFacility auth
+LogLevel info
+
+PermitRootLogin yes
+
+PrintMotd yes
```
Comment (jhoblitt) [inline roles/ssh/files/sshd_config]: Hmm.  I wonder if this might be messing with jenkins?  Seems unlikely but I'm grasping at straws.

File: roles/ssh/tasks/main.yml (hunk 1 above)
Comment (jmatt) [reply roles/ssh/tasks/main.yml:15]: That is how it was configured by ESXi. So I just kept it the same.

File: roles/ssh/files/sshd_config (hunk 2 above)
Comment (jmatt) [reply roles/ssh/files/sshd_config]: I'll disable it and we can find out.

**Comment by jmatt** [comment]
After using this to change `PrintMotd yes` - I'm going to add service restarts to both roles. Otherwise the configuration may not be applied.

**Comment by jmatt** [comment]
Added service restarts and changed `PrintMotd`.
